app/models/weet.js

```diff
@@ -0,0 +1,32 @@
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      userId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        field: 'user_id'
+      }
+    },
+    {
+      timestamps: true,
+      tableName: 'weets',
+      freezeTableName: true,
+      underscored: true
+    }
+  );
+  Weet.associate = models => {
+    Weet.belongsTo(models.User, { foreignKey: 'userId' });
```

Comment:
Nice, but add the `hasMany()` association to the `User` model as well since this is a one-to-many relation 👌